packages/amplify-category-custom/resources/package.json

```diff
@@ -9,11 +9,7 @@
   },
   "dependencies": {
     "@aws-amplify/cli-extensibility-helper": "^2.0.0",
-    "@aws-cdk/core": "^1.159.0",
-    "@aws-cdk/aws-iam": "^1.159.0",
-    "@aws-cdk/aws-sns-subscriptions": "^1.159.0",
-    "@aws-cdk/aws-sqs": "^1.159.0",
-    "@aws-cdk/aws-sns": "^1.159.0"
+    "aws-cdk-lib": "^2.41.0"
```

Comment:
let's bump `cli-extensibility-helper` to `^3.0.0`.
(and bump in `cli-extensibility-helper` itself, even if we don't migrate it yet in this PR)

Reply:
We should also bump version in  `amplify-category-custom/package.json` to `3.0.0`

Reply:
Should we bump this ```amplify-category-custom/package.json``` to ```3.0.0```. Or do it with lerna afterwards ?